packages/plexus/webpack-factory.js

```diff
@@ -195,13 +195,14 @@ function makeCommonProdConfig() {
 function makeWorkerConfig() {
   const layoutDir = join(__dirname, 'src/LayoutManager');
   const config = {
+    experiments: {
+      outputModule: true,
+    },
     output: {
       path: layoutDir,
       publicPath: '/',
       filename: '[name].bundled.js',
-      library: 'layout.worker.bundled',
```

Comment:
I remember there was some weirdness with this layout.worker - did you try running the deep dependency graphs after this change?

Reply:
Yeah, I tested rendering a DDG from a proxied-in Jaeger deployment that already had one, and it seems to load and render okay.